Jenkinsfile

```
@@ -691,19 +689,31 @@ def mageTarget(String context, String directory, String target) {
   }
 }
 
-def mageTargetWin(String context, String directory, String target) {
-  withGithubNotify(context: "${context}") {
-    withBeatsEnvWin() {
-      whenTrue(params.debug) {
-        dumpFilteredEnvironment()
-        dumpMageWin()
+def mageTargetWin(String context, String directory, String target, String label) {
+  return {
+    log(level: 'INFO', text: "context=${context} directory=${directory} target=${target} os=${label}")
+    node("immutable && ${label}"){
```

See https://github.com/elastic/beats/pull/18388/commits/8161f6c9c784769f0d02b5eab3598f8647b5a300

```suggestion
    def immutable = label.equals('windows-7-32-bit') ? 'windows-immutable-32-bit' : 'windows-immutable'
    node("${immutable} && ${label}"){
```